src/main/java/frc/robot/RobotContainer.java

```diff
@@ -57,6 +58,7 @@ public class RobotContainer {
       private final Feeder feeder = new Feeder();
       private Climber climber;
       private final IntakeSubsystem intakeSubsystem = new IntakeSubsystem();
+      private final CommandXboxController controller = new CommandXboxController(0);
```

Comment:
Move the ID of the Xbox controller to a constant (and probably shouldn't be 0).

Reply:
done